README.md

```diff
@@ -88,6 +88,60 @@ contains the OpenAPI Specification. Feel free to have a look
 at [this example spec file](https://github.com/masseelch/elk/tree/master/internal/simple/ent/openapo.json)
 and [the implementing server code](https://github.com/masseelch/elk/tree/master/internal/simple/ent/http).
 
+If you want to generate a client matching the spec as well, you can user the following function and call it after
+generating the spec in `entc.go`
+
+```go
+package main
+
+import (
+
+	"github.com/deepmap/oapi-codegen/pkg/codegen"
+	"github.com/deepmap/oapi-codegen/pkg/util"
+
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+func generateClient() {
+	swagger, err := util.LoadSwagger("./openapi.json")
+	if err != nil {
+		log.Fatalf("Failed to load swagger %v", err)
+	}
+
+	generated, err := codegen.Generate(swagger, "stub", codegen.Options{
+		GenerateChiServer:  false,
+		GenerateEchoServer: false,
+		GenerateClient:     true,
+		GenerateTypes:      true,
+		EmbedSpec:          false,
+		SkipFmt:            false,
+		SkipPrune:          false,
+		AliasTypes:         true,
```

Comment:
```suggestion
		GenerateClient:     true,
		GenerateTypes:      true,
		AliasTypes:         true,
```
I think we you dont need to explicitly set to false since that is the bool zero value